addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3011,6 +3012,37 @@ export class OdooEditor extends EventTarget {
                         );
                     }
                 }
+                if (ev.data === '`') {
+                    // We just inserted a backtick, check if there was another
+                    // one before.
+                    const range = getDeepRange(this.editable, { joinText: true, select: true });
+                    const textHasOpeningTick = /`.*`/.test(range.startContainer.textContent);
+                    if (textHasOpeningTick) {
+                        this.historyStep();
+                        let textNode = range.startContainer;
+                        if (range.startOffset !== textNode.textContent.length) {
+                            splitTextNode(textNode, range.startOffset);
+                            textNode = textNode.previousSibling;
+                        }
+                        const openingTickOffset = textNode.textContent.substring(0, textNode.textContent.length - 1).lastIndexOf('`');
```

Comment:
Why the substring ?

Reply:
So we don't match the closing backtick but the one just before it.

Reply:
I'm confused. What guarantees that the last character of the text node is the second backtick ?

Reply:
We just inserted it.

Reply:
That doesn't automatically make it the end of the textNode. Consider this:
```
<p>co`uc[]ou</p>
```
Now press `\``.

Reply:
Ok it is split right above.